images/runtime/php/template.base.Dockerfile

```diff
@@ -85,7 +85,11 @@ RUN set -eux; \
     fi
 
 # https://github.com/Imagick/imagick/issues/331
-RUN pecl install imagick && docker-php-ext-enable imagick
+# https://github.com/ihneo/php/pull/24/files
+RUN set -eux; \	
+    if [[ $PHP_VERSION != 8.3.* ]]; then \
+        pecl install imagick && docker-php-ext-enable imagick; \
+    fi
```

Comment:
how did we discover this issue - did we hit into error while trying to install it on 8.3?

Reply:
Yes while installing it.